orchestration/templates/cron-workflow.yaml

```diff
@@ -24,3 +27,26 @@ spec:
                 parameters:
                   - name: gcs-prefix
                     value: '{{ template "argo.timestamp" }}'
+      {{- if .Values.smokeTest.enable }}
+      - name: send-slack-notification
+        container:
+          image: curlimages/curl:7.70.0
+          env:
+            - name: WORKFLOW_ID
+              value: '{{ "{{workflow.name}}" }}'
+            - name: WORKFLOW_STATE
+              value: '{{ "{{workflow.status}}" }}'
+            - name: SLACK_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.smokeTest.slackUrl.secretName }}
+                  key: {{ .Values.smokeTest.slackUrl.secretKey }}
+          command: [curl]
+          args:
+            - -XPOST
+            - -H
+            - 'Content-type: application/json'
+            - -d
+            - '{"text": "Workflow $(WORKFLOW_ID) entered state: $(WORKFLOW_STATE)"}'
```

Comment:
We can do all sorts of fancy things w/ message formatting here, but I wasn't sure what would be useful. Any ideas?

Reply:
The `WORKFLOW_ID` would be something like `encode-ingest` right? If so, what you have makes sense to me.

Reply:
Hmm I'm wondering what other data points at the workflow level besides name and status?